lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
@njaffer What if you move the require out of engine.rb and put it in this block o' requires: 

https://github.com/projecthydra/sufia/blob/master/lib/sufia.rb#L1-L23

:question: